ci-operator/step-registry/configure-registry-storage/destroy-s3-cloudfront/configure-registry-storage-destroy-s3-cloudfront-ref.yaml

```diff
@@ -0,0 +1,16 @@
+ref:
+  as: configure-registry-storage-destroy-s3-cloudfront 
+  from: upi-installer
+  cli: latest
+  grace_period: 10m
+  commands: configure-registry-storage-destroy-s3-cloudfront-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  credentials:
+  - namespace: test-credentials
+    name: openshift-custom-mirror-registry
```

Comment:
is it required in destroy step?

Reply:
No need, I will remove it, just copy it directly